.github/workflows/deploy-site.yml

```diff
@@ -1,30 +0,0 @@
-name: Deploy website
-on:
-  release:
-    types: [published]
-    branches:
-      - master
-
-jobs:
-  build-and-deploy:
-    runs-on: ubuntu-latest
-    steps:
-    - name: checkout
-      uses: actions/checkout@master
-
-    - name: install
-      run: npm install
-
-    - name: build
-      run: npm run predeploy
-
-    - name: deploy
-      uses: peaceiris/actions-gh-pages@v2
-      env:
-        ACTIONS_DEPLOY_KEY: ${{ secrets.ACTIONS_DEPLOY_KEY }}
-        # PERSONAL_TOKEN: ${{ secrets.PERSONAL_TOKEN }}
-        # GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        PUBLISH_BRANCH: gh-pages
-        PUBLISH_DIR: ./_site
-      with:
-        emptyCommits: false
```

Comment:
This file should not remove

Reply:
There was an error while pushing with these files, github was denying push on workflow.
After some googling fixed by using PersonalAccessToken instead of password for git account.